scripts/migrate-framer-motion.js

```
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+import fs from 'fs';
+import path from 'path';
+import { execSync } from 'child_process';
+
+// Find all files that import from framer-motion
+const findFramerMotionImports = () => {
+  try {
+    const result = execSync(
+      'find /workspace/animata -type f -name "*.tsx" -o -name "*.ts" | grep -v "node_modules" | xargs grep -l "framer-motion"',
```

ESLint

[error] Strings must use doublequote.

(quotes)